src/java/org/apache/cassandra/tcm/Startup.java

```diff
@@ -298,25 +294,20 @@ public static void reinitializeWithClusterMetadata(String fileName, Function<Pro
 
         if (!metadata.isCMSMember(FBUtilities.getBroadcastAddressAndPort()))
             throw new IllegalStateException("When reinitializing with cluster metadata, we must be in the CMS");
+
         // can use local dc here since we know local host in the cms:
-        ClusterMetadata emptyFromSystemTables = emptyWithSchemaFromSystemTables(Collections.singleton(DatabaseDescriptor.getLocalDataCenter()));
-        metadata.schema.initializeKeyspaceInstances(DistributedSchema.empty());
-        metadata = metadata.forceEpoch(metadata.epoch.nextEpoch());
```

Comment:
Don't we need this anymore?

IIRC the reasoning when adding it was that someone dumps the metadata at epoch X - then, when we force it here, it should always be a new epoch

Reply:
You are right; i think I have removed this together with `metadata.schema.initializeKeyspaceInstances` by mistake.